perf/tuple.cc

```diff
@@ -134,7 +134,7 @@ class TestTuples {
 		for (size_t i = 0; i < NUM_TEST_TUPLES; i++) {
 			data[i] = box_tuple_new(format,
 						dataset[i].begin(),
-						dataset[i].end());
+						dataset[i].end(), true);
```

Comment:
Sadly, you can't change `box_tuple_new()`. It is a public C function, its signature is immutable. Same about `luaT_tuple_new()`.

Reply:
Oh, shoot, I didn’t pay attention, sorry.

Reply:
Added corresponding internal `_tuple_new_with_validation` functions that have a `validate` parameter.